dev-python/yoyo-migrations/yoyo-migrations-7.0.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="A database schema migration tool"
+HOMEPAGE="https://ollycope.com/software/yoyo"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
```

Comment:
I see they have at least one testfile included. Could you try implementing tests in this ebuild?

Reply:
unittest:
```
Ran 0 tests in 0.000s
```

pytest
```
no tests ran in 0.11 seconds
```